Makefile

```diff
@@ -206,16 +234,20 @@ operator-image:
 	rm -rf ${JUJUD_STAGING_DIR}
 
 push-operator-image: operator-image
+## push-operator-image: Push up the build operator image via docker
```

Comment:
* build and push up ..., Else people might think that they need to run `make operator-image` beforehand IMO

Reply:
Or we could just name the step which is used 
`## push-operator-image: run operator-image and push up .... `